### PR TITLE
feat: add trafficMetadata

### LINKED ISSUE #38: [Spec] Add a traffic.proto
**Current State:**
Traffic protocols will be referenced in multiple places within SPMS (e.g. in the workload definition and the performance test configuration). 

**Desired State:**
It will be good to define traffic specifications independently.

**Reference:**
When supporting gRPC, will the testURL be blank? Or be a function name? I think blank because it will just be a ping test.

_Originally posted by @leecalcote in https://github.com/layer5io/meshery/pull/1156_

### PR DESCRIPTION
**Pull request by kushthedude**
**Description**

This PR fixes #38

**Notes for Reviewers**

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by kushthedude** [comment]
This PR also fixes errors which were present in some proto contracts & follows a convention of snake case naming for variables in proto files.

File: traffic_metadata.proto [hunk 1]
```diff
@@ -0,0 +1,15 @@
+syntax = "proto3";
+
+package SMPS;
+
+option go_package = "spec;spec";
+
+message TrafficMetadata {
+  string load_balance_name = 1;
+  string load_balance_ip = 2;
+  string calico_hostendpoint = 3;
```
Comment (leecalcote) [inline traffic_metadata.proto:10]: I wonder if this should be generalized to just "host_endpoint".

Comment (kushthedude) [reply traffic_metadata.proto:10]: https://docs.projectcalico.org/introduction/ Calico is assigned a specific IP for traffic & security propose which is generalised by IBM & Red-Hat. I can add another field for host_endpoint if you say?

Comment (kushthedude) [reply traffic_metadata.proto]: @leecalcote fixed it!

**Pull request merged**